### DISCUSSION TITLE
Questions about numTriNonzeroIntAmbiguity and temporalCoherence

### DISCUSSION THREAD

**Discussion by david-ncu2019** [insarlab/MintPy#1301] [Q&A]
Dear authors of MintPy,

First, I would like to thank you all for creating such a great program for InSAR time-series analysis. In particular, the integration of MintPy with HYP3 has significantly sped up the processing time and saved a considerable amount of storage space.

However, during usage, I have encountered some issues that I am unable to interpret on my own. Below, I will present one of these issues and hope to receive suggestions from the authors or other users who may have encountered the same problem.

My input consists of ascending Sentinel-1 interferograms obtained from the HYP3 service (https://hyp3-docs.asf.alaska.edu/using/vertex/). My goal is to observe the development of land subsidence patterns in my study area.

After selecting the reference point, which is located far from the subsiding regions, I performed `quick_overview`. The `numTriNonzeroIntAmbiguity` output showed the following patterns:
![numTriNonzeroIntAmbiguity](https://github.com/user-attachments/assets/9cc3692b-dbb6-4a9b-8abf-5ff8bd86baf8)

Since the output from the HYP3 service did not contain `.conncomp` files, phase unwrapping error correction was not available during my processing. I then proceeded to `invert_network`, and below are the patterns from `temporalCoherence.h5`:
![spatialCoh_tempCoh_600dpi](https://github.com/user-attachments/assets/7440cfba-f834-470b-8566-70efdb0e3685)

As you can see, the temporal coherence values are separated into distinct zones, almost as if a "wall" is preventing the values from being smoothly continuous across the entire region.

Meanwhile, when I processed the descending interferograms obtained from the `hyp3-isce2` workflow (https://github.com/ASFHyP3/hyp3-isce2) with `insar_tops_burst`, this issue did not appear. I wonder if this issue is caused by the HYP3 Vertex service and how I can correct it.
![numTriNonzeroIntAmbiguity](https://github.com/user-attachments/assets/4366edd0-18c9-4b5e-bc0c-77c8a23bf1bb)
![desc_spatialCoh_tempCoh](https://github.com/user-attachments/assets/1b305b1e-8f15-4f47-a0a4-c3d85de07ac8)

**Comment by ditafaith** [comment]
Hi @david-ncu2019 
In my poor understanding, burst data can be better than gamma products due to mostly all images overlapping.
I just realized that mintpy can process burst. Is there any reference can be followed?

**Reply by david-ncu2019** [reply]
Hi @ditafaith ,

Thanks for your response.

To clarify, are you asking about the tutorials for processing the products of `insar_tops_burst` using MintPy?

**Reply by ditafaith** [reply]
Yes, it seems interest @david-ncu2019

**Reply by david-ncu2019** [reply]
Hi @ditafaith ,

You can find the tutorials here: https://hyp3-docs.asf.alaska.edu/tutorials/

**Reply by ditafaith** [reply]
That's great. Thanks @david-ncu2019
